src/options.ts

```diff
@@ -394,7 +394,7 @@ export interface ContainsOptions {
 }
 
 export interface IsAlphaOptions {
-  ignore?: string | RegExp;
+  ignore?: string[] | RegExp;
```

Comment:
```suggestion
  ignore?: string[] | string | RegExp;
```

should give maximum compatibility 👍